### PR TITLE
front: fix overlapping itinerary buttons

### LINKED ISSUE #6364: front: STDCM result panel overlapping itinerary buttons from stdcm config panel
### What happened?

image

When the screen is in low resolution the itinerary buttons are not wrapping and got overlap by the result view

### What did you expect to happen?

The button must wrap

### How can we reproduce it (as minimally and precisely as possible)?

Go to sdcm 
Resize the window 

### What operating system, browser and environment are you using?

- **Browser**: Firefox v120
- **OS**: Ubuntu 22.04
- **Env**: Local

### OSRD version (top right corner `Account` button > `Informations`)

9488d42

### PR DESCRIPTION
**Pull request by Omar-938**
- Itinerary.tsx : add flex wrap on line 113 and add margin bottom on line 130

close #6364

### CODE REVIEW

**Review by Yohh — COMMENTED** [review]
nice, good work

can you update the "add vias" column with responsive behavior as talked earlier?

**Comment by clarani** [comment]
Pro tip :wink: : You can add in the description of your PR the line `closes #[the number of your issue]`, in your case #6364. This way, the PR is linked directly to the board (see the screenshot below) 
image

File: front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx [hunk 1]
```diff
@@ -110,34 +110,35 @@ function Itinerary({ path }: ItineraryProps) {
         </div>
       )}
       {origin && destination && (
-        <div className="d-flex flex-row flex-wrap">
+
+        <div className="d-flex flex-column flex-sm-row flex-wrap"style={{ gap: '0.5rem' }}>
```
Comment (Caracol3) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx]: You should put the style in the css file or find the associated class in bootstrap if it exists

**Review by Caracol3 — CHANGES_REQUESTED** [review]
Good job for the PR!! Did you use ESlint and prettier in vscode? It helps to deal with formating issues

File: front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx (hunk 1 above)
Comment (Math-R) [reply front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx]: Yes, please avoid inline style

File: front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx [hunk 2]
```diff
@@ -110,34 +110,35 @@ function Itinerary({ path }: ItineraryProps) {
         </div>
       )}
       {origin && destination && (
-        <div className="d-flex flex-row">
+
+        <div className="d-flex flex-column flex-sm-row flex-wrap gap-2 btn-itinerary-gap">
```
Comment (Yohh) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx:114]:
```suggestion
        <div className="d-flex flex-column flex-sm-row flex-wrap btn-itinerary-gap">
```

**Review by Yohh — CHANGES_REQUESTED** [review]
please configure your linter and your formatter, you have many formatting issues that can't and will not be merged

**Comment by clarani** [comment]
Not relevant anymore

**Pull request closed without merging**